README.md

```diff
@@ -104,6 +104,7 @@ You can define whether you want each attribute shown by adjusting `PDPHitSection
 - Recommend components
   - Related Products
   - Frequently Bought Together
+  - Trending Products
```

Comment:
@diegolarraz this is the product details page section of the README, but Trending is on the Hompeage, can you update it to reflect this?